content/pages/burials-and-memorials/pre-need/form-10007-apply-for-eligibility.md

```diff
@@ -4,11 +4,12 @@ entryname: pre-need
 layout: page-react.html
 ---
 <div id="main">
-  <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
+  <nav aria-label="Breadcrumb" aria-live="polite" aria-relevant="additions text" class="va-nav-breadcrumbs js-visual"
+  id="va-breadcrumbs">
+    <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/burials-and-memorials/">Burials and Memorials</a></li>
-      <li><a href="/burials-and-memorials/pre-need/">Pre-need Eligibility Determination</a></li>
+      <li><a aria-current="page" href="/burials-and-memorials/pre-need/">Pre-need Eligibility Determination</a></li>
```

Comment:
Similar to above, this last page is a static content page, I think we want to add another item for the actual form page, if I'm understanding correctly.

Reply:
Tagging @mnorthuis on this comment. We discussed the IA strategy, and feel leaving the form breadcrumbs as they are is the best approach.

I added some markup to the `<nav>` and `<ul>` elements so the mobile-breadcrumbs.js script would apply our back-by-one transformation, and added the `aria-current="page"` attribute to the last breadcrumb link. That should be all the changes to static MD files.